### ISSUE TITLE
Pas de données MAJIC lues

### ISSUE THREAD

**Issue by TiphLB** [3liz/QgisCadastrePlugin#123]
Bonjour, 

Sous QGIS 2.16 j'ai installé la nouvelle version du Plugin Cadastre, ré-importé et re-chargé mes données en créant une BDD spatialite, tout se déroule bien mais rien faire, aucune recherche possible (commune / section / parcelle / propriétaire / ...). 
L'outil "Info Parcelle" ne fonctionne pas non plus ("La couche de parcelles n'a pas été trouvée dans le projet"), hors toutes mes couches se sont bien chargées après le chargement de données et ma couche parcelle est dans mon projet.

Des idées ?

Merci d'avance

**Comment by mdouchin** [comment]
Bonjour,

Cela doit être dû à un bug connu, qu'il faut que je résolve. Est-ce que la base de données spatialites est placée dans un répertoire dont un des éléments du chemin contient un espace ?

**Comment by TiphLB** [comment]
Oui, il est vrai qu'un élément du chemin contient un espace (aucun accent) mais ce chemin fonctionne très bien avec la version précédente du plugin que j'ai d'ailleurs ré-installé.

**Comment by MaelREBOUX** [comment]
@mdouchin c'est un problème d'import ou d'exploitation ?
